vg/draw/canvas.go

```diff
@@ -32,8 +32,47 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians, counterclockwise from
+	// the default drawing direction.
+	Rotation float64
+
+	// XAlign and YAlign specify the alignment of the text.
+	// Alignment and rotation are interrelated:
+	// when (XAlign, YAlign) = (XCenter,YCenter), the text is rotated about its center,
+	// (XLeft,YBottom) causes the text to rotate around its bottom left corner, etc.
+	XAlign XAlignment
+	YAlign YAlignment
 }
 
+// XAlignment specifies text alignment in the X direction. Three preset
+// options (XLeft, XCenter, and XRight) are available, but an arbitrary alignment
```

Comment:
This parenthetic statement is not necessary, they will see the default values in the godoc.